src/services/item/plugins/publication/validation/constants.ts

```diff
@@ -1,3 +1,5 @@
 export const PLUGIN_NAME = 'graasp-plugin-validation';
 
 export const IMAGE_CLASSIFIER_PREDICTION_THRESHOLD = 0.35;
+
+export const SVG_MIMETYPE = 'image/svg+xml';
```

Comment:
It already exists in sdk maybe we can reuse? https://github.com/graasp/graasp-sdk/blob/6cccfb970eaa55c93a5473323e14d93cbab9cb3e/src/mimeTypes/mimeTypes.ts#L38

Reply:
Yes good suggestion ! I saw and I didn't do that because I had to export it... but I think we can reuse it 🙌

Reply:
Oh I just realised that we didn't have to export the const of images ^^ So good suggestion 🙌